language/bytecode-verifier/invalid-mutations/src/bounds/code_unit.rs

```diff
@@ -292,8 +292,8 @@ impl<'a> ApplyCodeUnitBoundsContext<'a> {
                     FreezeRef | Pop | Ret | LdConst(_) | LdTrue | LdFalse | ReadRef | WriteRef
                     | Add | Sub | Mul | Mod | Div | BitOr | BitAnd | Xor | Or | And | Not | Eq
                     | Neq | Lt | Gt | Le | Ge | Abort | GetTxnGasUnitPrice | GetTxnMaxGasUnits
-                    | GetGasRemaining | GetTxnSenderAddress | CreateAccount
-                    | GetTxnSequenceNumber | GetTxnPublicKey => {
+                    | GetGasRemaining | GetTxnSenderAddress | GetTxnSequenceNumber
+                    | CreateAccountDeprecated | GetTxnPublicKey => {
```

Comment:
Why do we need this deprecated code? or is this just for WIP?

Reply:
You could batch them out, but I'm not sure it would be that much different. The only real effect is making two breaking changes than one :P (obviously these things are staged so it will just be one but)

I think in our current state we should just delete the bytecodes when they change. The file format should be expected to be highly volatile at this point